TombEngine/Game/effects/effects.cpp

```diff
@@ -1574,8 +1606,21 @@ void ProcessEffects(ItemInfo* item)
 				TriggerRocketSmoke(pos.x, pos.y, pos.z, 0);
 			break;
 
+		case EffectType::ElectricDeath:
+			if (TestProbability(1 / 1.0f))
+				TriggerElectricSpark(&GameVector(pos.x, pos.y, pos.z, item->RoomNumber),
+					EulerAngles(0, Random::GenerateAngle(ANGLE(0), ANGLE(359)), 0), 2);
+			if (TestProbability(1 / 1.0f))
+				TriggerFireFlame(pos.x, pos.y, pos.z, TestProbability(1 / 10.0f) ? FlameType::Color : FlameType::Color, Vector3(40, 120, 255), Vector3(53, 215, 255));
+			break;
+
+		case EffectType::LaserDeath:
+			if (TestProbability(1 / 1.0f))
+				TriggerFireFlame(pos.x, pos.y, pos.z, TestProbability(1 / 10.0f) ? FlameType::Color : FlameType::Color, Vector3(255, 130, 40), Vector3(160, 30, 0));
+			break;
+
 		case EffectType::Smoke:
-			if (TestProbability(1 / 8.0f))
```

Comment:
Why this was changed so much?

Reply:
oh, I changed it to 64.0f. Isn't 8.0f too much smoke?

Reply:
No, I think it's okay.